README.md

```diff
@@ -8,9 +8,19 @@ npm i
 ```
 in the repository root
 
+You also need to install rethinkdb by downloading it and running it from command prompt
+
+Please remember to insert the uservoice details into the config as well as bot token and role ids for Admins/Mods.
```

Comment:
Before running the bot there is a few items to be replaced, they are as follows, replace the uservoice details with your info's, the discord bot token, and role ids (from the server that you are wanting to run the bot in, you need to make sure you have these roles added to your user or else you wouldn't be able to use the bot!).